README.md

```diff
@@ -135,6 +187,15 @@ The tint color of the progress bar track.
 | ------ | -------- |
 | string | No       |
 
+### `isIndeterminate`
```

Comment:
I see you added this new, Windows-specific prop, and that it is shown in the image in the PR description. I can't see it used in code, though. Where is the file where it is used?